eng/Versions.props

```diff
@@ -18,7 +18,6 @@
     <MajorVersion>7</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
-    <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
```

Comment:
Our Razor language server binaries were being generated with the `7.0.0-preview` format. Since we're not using a preview SDK anymore, I think we can emit it

Reply:
Fair! I'll remove that bit.